fe/fe-core/src/test/java/org/apache/doris/planner/QueryPlanTest.java

```diff
@@ -1366,23 +1366,15 @@ public void testRuntimeFilterMode() throws Exception {
 
         queryStr = "explain select * from jointest t2, jointest t1 where t1.k1 <=> t2.k1";
         Deencapsulation.setField(connectContext.getSessionVariable(), "runtimeFilterMode", "LOCAL");
-        Deencapsulation.setField(connectContext.getSessionVariable(), "runtimeFilterType", 7);
+        Deencapsulation.setField(connectContext.getSessionVariable(), "runtimeFilterType", 15);
```

Comment:
What is the main purpose of this single test?
Type=15 is meaningless, writing this way will give users a wrong guide.
Can I change the type? Such as 8

Reply:
> Can I change the type? Such as 8

You can.
origin test, 7 means: IN,BLOOM_FILTER,MIN_MAX
new test, 15 means: IN,BLOOM_FILTER,MIN_MAX,IN_OR_BLOOM_FILTER

Already test `8` in other test